api/turing/cluster/controller.go

```diff
@@ -517,44 +517,23 @@ func (c *controller) GetKnativeServiceURL(ctx context.Context, svcName string, n
 	return url
 }
 
-// ApplyPersistentVolumeClaim creates a PVC in the given namespace.
-// If the PVC already exists, it will update the existing PVC.
-func (c *controller) ApplyPersistentVolumeClaim(
+// DeleteStatefulSetPersistentVolumeClaims deletes all PVCs belonging to the specified stateful set in the given
+// namespace.
+func (c *controller) DeleteStatefulSetPersistentVolumeClaims(
```

Comment:
nit: could be more generic as `DeletePVC` and take in `listOptions` instead of `statefulSetName`

Reply:
Alright sounds good; will refactor this.